### PR TITLE
PDG-11

### PR DESCRIPTION
**Pull request by lognaume**

### CODE REVIEW

File: server/src/main/java/ch/heig/services/GameService.java [hunk 1]
```diff
@@ -41,19 +56,224 @@ public int subscribeToGame(Token token) {
...
+        gameRepository.save(gameEntity);
+
+        return LetterToRetrieve;
+    }
+
+    public int verifyWord(Play play, long gameId, long userId) {
+        GameEntity gameEntity = gameRepository.findOne(gameId);
+
+        if (!gameEntity.getInitialized()) {
+            return -100;
```
Comment (Rhod3) [inline server/src/main/java/ch/heig/services/GameService.java]: Change magic numbers

**Comment by Rhod3** [comment]
## Bugs
* In the statistic page, expand "Ret..." button to fully display button text
* In the statistic page, "Ret..." button does nothing but launching an exception to the client
* In the statistic page, statistics doesn't display ?

**Comment by lognaume** [comment]
@Rhod3 fixed

File: server/src/main/java/ch/heig/services/GameService.java (hunk 1 above)
Comment (Farenjihn) [reply server/src/main/java/ch/heig/services/GameService.java]: Delet this.

File: client/src/main/java/ch/heig/controllers/LoggedInController.java [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Class description
```
Comment (Oceanos1) [inline client/src/main/java/ch/heig/controllers/LoggedInController.java]: javadoc

File: server/src/main/java/ch/heig/services/GameService.java (hunk 1 above)
Comment (Oceanos1) [reply server/src/main/java/ch/heig/services/GameService.java]: burn it with fire

File: server/src/main/java/ch/heig/services/LangService.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package ch.heig.services;
+
+import ch.heig.language.Language;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
```
Comment (Oceanos1) [inline server/src/main/java/ch/heig/services/LangService.java]: Javadoc

File: server/src/main/java/ch/heig/services/LetterService.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+import java.io.File;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
```
Comment (Oceanos1) [inline server/src/main/java/ch/heig/services/LetterService.java]: javadoc

**Pull request merged**